src/overlays/actors/ovl_En_Dy_Extra/z_en_dy_extra.c

```diff
@@ -1,15 +1,27 @@
+/*
+ * File: z_en_dy_extra.c
+ * Overlay: ovl_En_Dy_Extra
+ * Description: Spiral Beams (Great Fairy Fountains)
+ */
+
 #include "z_en_dy_extra.h"
+#include "vt.h"
 
 #define FLAGS 0x00000030
 
 #define THIS ((EnDyExtra*)thisx)
 
+extern UNK_TYPE D_0601BFB0;
+extern Gfx* D_0601C160;
+
```

Comment:
not a huge deal but i like to keep the externs underneath all the data and stuff